### DISCUSSION TITLE
MixxxPiBeta64v3

### DISCUSSION THREAD

**Discussion by fayaaz** [fayaaz/mixxx-pi-gen#46] [General]
- Update to Debian 11
- Mixxx 2.4 alpha version  [4e90c681b61c4c191b7480b8c872d16e881cdeb5](https://github.com/mixxxdj/mixxx/commit/4e90c681b61c4c191b7480b8c872d16e881cdeb5)
Be warned this is a Mixxx alpha version and may have bugs
- /code/ directory deleted to shrink disk image size.
- Tested on a Raspberry Pi 400
- Added .deb of built Mixxx to release alongside disk image

<hr /><em>This discussion was created from the release <a href='https://github.com/fayaaz/mixxx-pi-gen/releases/tag/v0.7.0'>MixxxPi64v3</a>.</em>

**Comment by mttm22** [comment]
Hi, I tried it on Raspberry Pi 4B, works fine.
The only problem is with the 7" screen resolution, too small.
Is it possible to install TimeWasterNL's new responsive skin from this image?
Sorry for the question but I am a noob

**Reply by fayaaz** [reply]
You can copy the skin to the raspberry pi:
https://github.com/fayaaz/mixxx-pi-gen/issues/50#issuecomment-1230806115

I will look to add it in a future release, but it would be nice if it was added to Mixxx as an official skin!
